modules/rostests/kmtests/kmtest_drv/kmtest_fsminifilter.c

```diff
@@ -470,9 +472,13 @@ KmtFilterRegisterCallbacks(
         Callbacks[i].PostOperation = FilterPostOperation;
     }
 
+    /* FIXME: Useless? See also CallbacksCount value/uses */
```

Comment:
This isn't useless. Windows needs the array to be properly terminated.
What you can't see here is that the array pointer will be set in in the registration struct before the filter is registered with the fltmgr. I just haven't gotten around to that part yet.

Unfortunately, you're submitting patches to unfinished and untested code. I do have some local changes, and will get around to finishing this eventually :/

Reply:
I know this code is unfinished. I assumed this line is intentional. Thanks for the review/explanation.
`FIXME()` removed.